libs/sdk-ui-charts/src/charts/_base/BaseChart.tsx

```diff
@@ -21,6 +21,7 @@ import {
 import { getSanitizedStackingConfig } from "../_commons/sanitizeStacking";
 import { ITheme } from "@gooddata/sdk-backend-spi";
 import { ThemeContextProvider } from "@gooddata/sdk-ui-theme-provider";
+import { ChartTransformation } from "../../highcharts";
```

Comment:
Nitpick: please merge this import with the one on line 4

Reply:
Done: https://github.com/gooddata/gooddata-ui-sdk/pull/1277/commits/e8e4b785212e297be75a8355b4b6f52dcbf46dbb